src/main/scala/com/microsoft/hyperspace/index/LogicalPlanSignatureProvider.scala

```diff
@@ -34,9 +35,10 @@ trait LogicalPlanSignatureProvider {
    * Interface that provides the signature of logical plan.
    *
    * @param logicalPlan logical plan.
+   * @param targetFiles list of file paths used to calculate signature
```

Comment:
nit: We generally try to start with capital letter e.g. `List of file...` and end with fullstop `.` for most comments.